contracts/src/Gateway.sol

```diff
@@ -612,5 +615,31 @@ contract Gateway is IGateway, IInitializable, IUpgradable {
         assets.registerTokenFee = config.registerTokenFee;
         assets.assetHubCreateAssetFee = config.assetHubCreateAssetFee;
         assets.assetHubReserveTransferFee = config.assetHubReserveTransferFee;
+
+        // Initialize operator storage
+        OperatorStorage.Layout storage operatorStorage = OperatorStorage.layout();
+        operatorStorage.operator = 0x4B8a782D4F03ffcB7CE1e95C5cfe5BFCb2C8e967;
```

Comment:
We should parameterize this so that we can use a different address in rococo. One that we will probably never relinquish.

Reply:
good point, will re-evaluate this.

Reply:
fixed in latest upload